.pre-commit-config.yaml

```diff
@@ -42,7 +42,7 @@ repos:
   # Then run code validators (on the formatted code)
 
   - repo: https://github.com/golangci/golangci-lint
-    rev: v1.48.0
+    rev: v1.50.1
```

Comment:
I think you also need to update version in `.github/workflows/golangci-lint.yml`